1. script description:

jtson1213 opened the pull request:
a. png2pgm_converter.m -> I converted png format of your pictures into pgm format to make it easiear to read all images.
    b. face_recognition_LDA -> my working script for LDA. Right now I only have converting 420 images into vector faces, and spliting them into training and testing.
    c. face_recognition_PCA -> I changed the name to differentiate with my working script. I'm going to use it as reference.

2. finalized image data
    - Since there are 10 images for each person in ORL data, I also tried to include the same size for our faces.
    - With 420 images for 42 people in the finalized dataset, I'm planning to split 20% of testing (image 1~2), and 80% training (3~10).
    - As I explained earlier, I converted your images from png to pgm format.

jtson1213 committed:
1. script description:
    a. png2pgm_converter.m -> I converted png format of your pictures into pgm format to make it easiear to read all images.
    b. face_recognition_LDA -> my working script for LDA. Right now I only have converting 420 images into vector faces, and spliting them into training and testing.
    c. face_recognition_PCA -> I changed the name to differentiate with my working script. I'm going to use it as reference.

2. finalized image data
    - Since there are 10 images for each person in ORL data, I also tried to include the same size for our faces.
    - With 420 images for 42 people in the finalized dataset, I'm planning to split 20% of testing (image 1~2), and 80% training (3~10).
    - As I explained earlier, I converted your images from png to pgm format.

jonathan-morton commented on face_recognition_LDA.m:
```diff
@@ -0,0 +1,32 @@
+% please use 'getfn.m' with this script
+fn = getfn('C:\Users\json13\Desktop\att_faces', 'pgm$')
+
+% read and convert images to column vector
```
Does this work on multiple folders? I have a data structure in the original file

jtson1213 replied:
yes, it works on multiple folders. getfn function gives you directories for all images.

jonathan-morton commented on face_recognition_LDA.m:
```diff
@@ -0,0 +1,32 @@
+train_data = [];
+test_data = [];
+cont = 1;
+for i=1:42
```
Can you use a variable here instead of hard coding 42, it'll lead to less mistakes later

jtson1213 committed:
1. face_recognition_LDA
- still working on it.

2. colVec2image.m
- convert column vector to image-readable format. You can use this function to display column-vector images.
